css/styles.css

```diff
@@ -68,6 +82,15 @@ button {
   letter-spacing: 1.5px;
 }
 
+ul {
+  list-style-type: none;
+  margin: 0;
```

Comment:
you can either remove the `margin` prop here or better to add `width: 100%` to the `.row` class in your `layout.css` file